common/network-common/src/main/java/org/apache/spark/network/server/OneForOneStreamManager.java

```diff
@@ -95,6 +95,14 @@ public ManagedBuffer getChunk(long streamId, int chunkIndex) {
   }
 
   @Override
+  public ManagedBuffer openStream(String streamChunkId) {
+    String[] array = streamChunkId.split("_");
```

Comment:
Instead of spread the parsing logic, it is better to externalize this into a pair of methods - one to create streamChunkId given streamId and chunkIndex and another to retrieve it.
If we have to change delimiter or add other logic, it will be more easier to manage the change.